### ISSUE TITLE
go build seems to fail

### ISSUE THREAD

**Issue by jaltek** [fourcube/goiban-service#11]
Hi,

I'm getting the following "issue" when trying to ```go build``` the service:

```
$ go build
# github.com/armon/go-metrics
../../armon/go-metrics/inmem_endpoint.go:72: undefined: sort.Slice
../../armon/go-metrics/inmem_endpoint.go:86: undefined: sort.Slice
../../armon/go-metrics/inmem_endpoint.go:113: undefined: sort.Slice
$ echo $?
2
```
(Exit code 2)
Maybe an upstream issue?

```
$ go version
go version go1.6.2 linux/amd64
```

**Comment by jaltek** [comment]
Ok, seems to be a problem with go 1.6. and go 1.7 according to this issue and PR:

https://github.com/armon/go-metrics/issues/59
https://github.com/armon/go-metrics/pull/60

Any chance to build the service without the metrics support?

**Comment by fourcube** [comment]
Thanks for tracking this down. I'll create a build tag and some instructions which will allow you to excludes the metrics functionality.

**Comment by fourcube** [comment]
Fixed by [#490dded](https://github.com/fourcube/goiban-service/commit/490ddedd4279224978d68c46a61a741157e70310). 

You can run 

```
$ go build -tags no_metrics
```

to create a build without metrics support.

**Comment by jaltek** [comment]
Awesome. Works like a charm. Thanks a lot!
